browser/src/control/Control.Toolbar.js

```diff
@@ -210,6 +210,44 @@ function getConditionalFormatMenuElementsImpl(more, type, count, unoCommand, jsd
 	return table;
 }
 
+function sendUnoCommandWithSlideLayoutParam(num, unoCommand, jsdialogDropdown) {
```

Comment:
We should add all new code in TypeScript. Best to put single feature-related code in separate file.
I understand it was quicker here as you had some base to be inspired - but just note for future.
We can later convert that - no need to do in this PR :)